coverage.sh

```diff
@@ -32,6 +32,15 @@ fi
 ./sizes
 ./constants
 ./multi
+for i in $(./hashsum | tail -n +3 | awk '{print $1}' | sort); do echo -n "$i: " && ./hashsum $i testprof/test.key ; done > hashsum_tv.txt
```

Comment:
I see. IMO not worth blocking this PR. 

BTW shouldn't it be better tu utilize `hash_file` and/or `hash_filehandle` in `hashsum.c`?